app/views/propertyrepresentation/manage/manageAgent.scala.html

```diff
@@ -41,7 +42,7 @@
 
 @mainLayout(
     pageTitle = messages("propertyRepresentation.manageAgent.title"),
-    backUri = Some("#")
+    backUri = Some(controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url)
```

Comment:
why PaginationParameters(1, 15) ?

Reply:
That is default page size on Page Load When Search Functionality Applies
Page load from Back button, we need to reload the page with Default Pagination and Filter Parameters

Reply:
ok - you can use DefaultPaginationParameters